src/components/FormComponent/_Select.tsx

```diff
@@ -0,0 +1,31 @@
+import MenuItem from '@mui/material/MenuItem';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { ChangeEvent, useState } from 'react';
+
+export type _SelectProps = Omit<TextFieldProps, 'select' | 'type'> & {
+  options?: { label: string; value: string }[];
+};
+
+export const _Select = (props: _SelectProps) => {
+  const [value, setValue] = useState('');
+
+  const onChange = (e: ChangeEvent<HTMLInputElement>) => setValue(e.target.value);
+
+  return (
+    <TextField
```

Comment:
Why a TextField to render a `Select box` ?

Reply:
No reason really, just personal preference because is less code and the label is handled by one component. It still uses Mui Select component under the hood, but i can change it to a Select